src/containerapp/azext_containerapp/_clients.py

```diff
@@ -128,8 +128,26 @@ def poll_results(cmd, request_url):  # pylint: disable=inconsistent-return-state
 
     animation.flush()
     if r.text:
-        return json.loads(r.text)
-
+        clean_json = clean_sensitive_values(r, show_sensitive_values)
```

Comment:
`poll_result` is used by many Clients, should not add  `clean_sensitive_values ` which only for containerappClient in this function.